psi4/driver/driver_cbs.py

```diff
@@ -215,7 +221,13 @@ def xtpl_highest_1(functionname: str, zHI: int, valueHI: float, verbose: bool =
         return valueHI
 
 
-def scf_xtpl_helgaker_2(functionname: str, zLO: int, valueLO: float, zHI: int, valueHI: float, verbose: bool = True, alpha: float = None):
+def xtpl_exponential_2(functionname: str,
```

Comment:
I wanted to have the names of the extrapolation functions to correspond to the actual formula being used for the extrapolation (exponential, expsqrt, power), but not break backwards compatibility. Hence, there are 6 "new" generic functions, and the previous "named" functions just set a default alpha.

The idea is that if the user provides an alpha, they might want to specify what math function they want to use, rather than having to figure out if `expsqrt` is "Karton" or "Truhlar"...